Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:18.04
-
+FROM alpine:3.13.2 AS builder
```

Comment:
This is needed just for the build, right? So yeah we don't need PARITY here

Reply:
I suppos we'll get rid of Parity changes after the OE upgrade. IMO need to have a separate backlog for Network